File: app/src/main/java/com/startngproject/aboyakotlin/MainActivity.java
package com.startngproject.aboyakotlin;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView nameTv, userNameTv, emailTv, trackTv;

        String name, username, email, tracks;

        name = "Udo Bassey Nkereuwem";
        username = "Aboya";
        email = "basseynk2@gmail.com";
        tracks = "Mobile";


        nameTv = findViewById(R.id.textViewName);
        userNameTv = findViewById(R.id.textViewUsername);
        emailTv = findViewById(R.id.textViewEmail);
        trackTv = findViewById(R.id.textViewTracks);

        nameTv.setText("Name: " + name);
        userNameTv.setText("Username: " + username);
        emailTv.setText("Email: " + email);
        trackTv.setText("Track(s): " + tracks);
    }
}

File: app/src/main/java/com/startngproject/aboyakotlin/MainActivity.kt
package com.startngproject.aboyakotlin

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val name = "Udo Bassey Nkereuwem"
        val username: String = "Aboya1417"
        val email: String = "basseynk2@gmail.com"
        val tracks: String = "Mobile"
        val nameTv: TextView = findViewById(R.id.textViewName)
        val userNameTv: TextView = findViewById(R.id.textViewUsername)
        val emailTv: TextView = findViewById(R.id.textViewEmail)
        val trackTv: TextView = this.findViewById(R.id.textViewTracks)
        nameTv.text = "Name: $name"
        userNameTv.text = "Username: $username"
        emailTv.text = "Email: $email"
        trackTv.text = "Track(s): $tracks"
    }
}